app/src/main/java/com/jerboa/ui/components/common/MarkdownHelper.kt

```diff
@@ -165,25 +165,41 @@ object MarkdownHelper {
         color: Color = Color.Unspecified,
         onClick: (() -> Unit)? = null,
         onLongClick: (() -> Unit)? = null,
+        maxLines: Int? = null,
+        style: TextStyle? = null,
+        removeImages: Boolean = false,
     ) {
-        val style = MaterialTheme.typography.bodyLarge
         val defaultColor: Color = LocalContentColor.current.copy(alpha = LocalContentAlpha.current)
-
+        val typographyStyle = style ?: MaterialTheme.typography.bodyLarge
+        val preppedMarkdown = if (removeImages) {
+            // use regexp to parse out any markdown image tags
+            val regexp = Regex("!\\[(.*?)]\\(.*?\\)")
```

Comment:
This seems a bit hacky. I was thinking more along the lines of having a second Markdown renderer (mostly copied from the first), that doesn't load coil / use images.

Thoughts @twizmwazin ?

Reply:
Yeah I agree, thanks for addressing it @vijaykramesh